03 Writing Algorithms/24 Reality Modeling/08 Options Models/03 Exercise/05 Model Structure.html

```diff
@@ -14,21 +13,34 @@
             option.Symbol,
             option.LocalTime.ConvertToUtc(option.Exchange.TimeZone),
             OrderStatus.Filled,
-            GetOrderDirection(order.Quantity),
+            Extensions.GetOrderDirection(order.Quantity),
```

Comment:
Can we not replace it for `order.Quantity.GetOrderDirection()`?

Reply:
I don't think it has that https://github.com/QuantConnect/Lean/blob/fb383dc73c43b4744ff573a8f69a5a52dd462504/Common/Extensions.cs#L3373-L3400